lib/graphql-anycable.rb

```diff
@@ -11,33 +11,50 @@
 
 module GraphQL
   module AnyCable
-    def self.use(schema, **opts)
-      schema.use(GraphQL::Subscriptions::AnyCableSubscriptions, **opts)
-    end
+    class << self
+      def use(schema, **opts)
+        schema.use(GraphQL::Subscriptions::AnyCableSubscriptions, **opts)
+      end
 
-    def self.stats(**opts)
-      Stats.new(**opts).collect
-    end
+      def stats(**opts)
+        Stats.new(**opts).collect
+      end
+
+      def redis=(connector)
+        @redis_connector = if connector.is_a?(::Proc)
+          connector
+        else
+          ->(&block) { block.call connector }
+        end
+      end
+
+      def with_redis(&block)
+        @redis_connector || default_redis_connector
+        @redis_connector.call { |conn| block.call(conn) }
```

Comment:
Nitpick: I believe that just passing block as a block is more concise (and we don't need to create a wrapper block that just calls another block)
```suggestion
        @redis_connector.call(&block)
```